src/locales/sp/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,637 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
@Cegard look at this PR #144 where a corrected few errors on your preview file structure, note this:
the code of spanish is `es` not `sp` : http://www.lingoes.net/en/translator/langcode.htm
so the folder should be src/locales/es/LC_MESSAGES/messages.po as I corrected previously